client/src/components/NavBar.jsx

```diff
@@ -1,32 +1,31 @@
 import { BsPersonCircle, BsGeoAlt, BsTrophy } from "react-icons/bs";
-import { NavLink } from "react-router-dom";
+import { Link, NavLink } from "react-router-dom";
 import "../styles/NavBar.css";
 
 function Navbar() {
   return (
     <nav className="navbar">
-      <img
-        className="logo"
-        src="./src/assets/images/logo.webp"
-        alt="logo pixhunter"
-      />
+      <Link to="/">
+        <img
+          src="./src/assets/images/logo.webp"
+          alt="logo pixhunter"
+          className="logo"
+        />
+      </Link>
       <ul className="navLists">
         <li className="navList">
           <NavLink className="link-navIcon" to="/position">
             <BsGeoAlt className="navIcon" />
-            <p className="navText">Page d'accueil</p>
```

Comment:
Why have the texts disappeared?

Reply:
We used pseudo-elements in CSS to display the text. It's not ideal ?